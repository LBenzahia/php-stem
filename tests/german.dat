a
aa
aalglatten
aargau
aargauer
aasen
aasgeruch
aashöllen
ab
abaddon
abarten
abbeisst
abbild
abbildung
abbiss
abbrechen
abbruch
abend
abendbrot
abenddämmerung
abenddunklen
abende
abendessen
abendherren
abendhimmel
abendländischem
abendländischen
abendlichen
abendlichte
abendluft
abendmahl
abendröte
abends
abendschein
abendscheine
abendschöppchen
abendsonne
abendstern
abendtanz
abendwind
abendzeit
abenteuer
abenteuerlich
abenteuerliche
abenteuerlichem
abenteuerliches
abenteuern
abenteuers
abenteuerspielplätzen
abenteurer
aber
aberglaube
aberglauben
abermalige
abermaligen
abermals
abertausend
aberwitzigen
abfahrt
abfall
abfallen
abfällige
abfallkorb
abfällsel
abfällt
abfeuernd
abfielen
abfinden
abflughalle
abfressen
abfuhr
abführung
abgab
abgabe
abgaben
abgang
abge
abgebe
abgeben
abgebildet
abgebracht
abgebrannten
abgebrochen
abgebrochene
abgebrochenes
abgebürstet
abgedient
abgedruckt
abgefallen
abgefangen
abgefertigt
abgeführt
abgefunden
abgegebene
abgegrenzt
abgegriffene
abgehalten
abgehandelt
abgehandelten
abgehängen
abgehärmte
abgehärteten
abgehauen
abgehen
abgeholt
abgehört
abgeht
abgekauft
abgekommen
abgeladen
abgelagertes
abgelassen
abgelaufen
abgelebte
abgelegenen
abgelegt
abgelehnt
abgeleitet
abgeleitete
abgeleiteten
abgeleitetes
abgelenkt
abgeleugnet
abgelöst
abgemagert
abgemessenheit
abgenagten
abgeneigt
abgenommen
abgenutzten
abgeödet
abgeödete
abgeordnetenversammlung
abgeordneter
abgepfändet
abgepflückt
abgeplatzt
abgeraten
abgerieben
abgerissen
abgerissenen
abgesandt
abgeschabtes
abgescheuert
abgeschieden
abgeschiedenen
abgeschlagen
abgeschliffen
abgeschmolzen
abgeschnitsabgeschmtsabgeschgefalle