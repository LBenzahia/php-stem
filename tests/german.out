a
aa
aalglatt
aargau
aargau
aas
aasgeruch
aasholl
ab
abaddon
abart
abbeisst
abbilduessildureas
aduraas
abend
abendbrot
abenddammer
abenddunkl
abend
abendessildendherr
abendhimmel
abendland
abendland
abend
abendlicht
abendluft
abendmahl
abendrot
abend
abendschein
abendschein
abendschoppas
abendsonn
abend
abendtanz
abendwind
abendzeit
abenteu
abenteu
abenteu
abenteu
abenteu
abenteu
abenteu
abenteuerspielp